### DISCUSSION TITLE
How to sum (total) numeric values in a column

### DISCUSSION THREAD

**Discussion by AndrewBlumhardt** [rod-trent/MustLearnKQL#21] [General]
I am having a brain freeze today. If you have a bunch of rows with a numeric column, let's say count. How can you add together all of those values into a total? For example, each row has a failure count and I want to display total failures.

**Comment by theheatDK** [comment]
Maybe by using sum?

https://docs.microsoft.com/en-us/azure/data-explorer/kusto/query/sum-aggfunction

```kusto
StormEvents 
| summarize EventCount=count(), TotalDeathCases = sum(DeathsDirect) by State 

```
Regards Peter

**Comment by rod-trent** [comment]
Something like:

AzureActivity
| summarize count() by ActivityStatusValue
| summarize sum(count_)
